torch/backends/quantized/__init__.py

```diff
@@ -22,7 +22,7 @@ def __init__(self, m, name):
     def __getattr__(self, attr):
         return self.m.__getattribute__(attr)
 
-    engine = ContextProp(torch._C._get_preferred_engine, torch._C._set_preferred_engine)
+    engine = ContextProp(torch._C._get_qengine, torch._C._set_qengine)
```

Comment:
btw, I was chatting with Greg that polluting the main namespace with fbgemm/qnnpack is not nice. We could just expose it to python via strings (or None for empty). But it can be done in a separate diff